scripts/example.env

```diff
@@ -0,0 +1,5 @@
+DOMAIN= # It is obtained by analyzing the "domain" field of the manifest database in the "file" table.
```

Comment:
As you're adding a .env file, the README should be updated accordingly, so this doesn't become hidden knowledge :)

Reply:
Documentation added